src/assets/js/main.js

```diff
@@ -89,3 +89,7 @@
         });
     }
 })();
+
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Can you also move this into post.html as an inline script? This causes an error on non-blog post pages now.

Reply:
Done